tests/unit/matrix/test_measure.py

```diff
@@ -127,6 +123,71 @@ def it_provides_access_to_various_measure_objects(
         MeasureCls_.assert_called_once_with(dimensions_, measures, cube_measures_)
         assert measure is measure_
 
+    @pytest.mark.parametrize(
+        "measure_prop_name, MeasureCls, proportion_type, base_type",
+        (
+            (
+                "column_proportion_variances",
+                _ProportionVariances,
```

Comment:
I think the `MeasureCls` should not be a param of the test considering that it's always `_ProportionVariances`